book/enterprise-contract.md

```diff
@@ -0,0 +1,159 @@
+ec-cli refer to the
+[documentation](https://hacbs-contract.github.io/ec-cli/main/reference.html)
+and the [code](https://github.com/hacbs-contract/ec-cli).
+
+### The EC Task Definition
+
+The EC Task Definition defines how the ec-cli command should be run in a
+Tekton task. It handles the task inputs and outputs and calls the ec-cli as
+needed to perform the EC validation.
+
+The task is defined
+[here](https://github.com/hacbs-contract/ec-cli/blob/main/deployments/verify-enterprise-contract.yaml),
+and is available in a task bundle at _todo: where??_.
+
+### The EC Policy CRD
+
+The ECP CRD defines a Kubernetes CR which is used to hold the configuration
+needed for running a specific instance of the Enterprise Contract. This
+includes the public key required to verifying signatures, the list of policy
+and data sources, and any other required configuration.
+
+You can view the source code for the ECP CRD
+[here](https://github.com/hacbs-contract/enterprise-contract-controller) and
+see its documentation [here](https://hacbs-contract.github.io/ecc/main/). See
+also the related [API
+Reference](https://redhat-appstudio.github.io/book/ref/enterprise-contract.html)
+
+### EC Policies
+
+The reference set of rego policies created for the HACBS MVP are defined
+[here](https://github.com/hacbs-contract/ec-policies/) and are documented
+[here](https://hacbs-contract.github.io/ec-policies/). In includes rules for a
+range of different policies that are considered important for HACBS.
+
+There are conftest bundles containing the latest version of these policies
+available in [quay.io
+here](https://quay.io/repository/hacbs-contract/ec-release-policy?tab=tags).
+
+
+Related Components
+------------------
+
+### Tekton Chains
+
+Tekton Chains is a dependency for EC since EC works by examining attestations
+created by Tekton Chains when HACBS build pipelines are running.
+
+For more information on Tekton Chains refer to the
+[documentation](https://tekton.dev/docs/chains/), and the GitOps configuration
+[here](https://github.com/openshift-pipelines/pipeline-service/tree/main/operator/gitops/argocd/tekton-chains).
+
+### The Release Pipeline
+
+The HACBS Release Pipeline contains an instance of the EC Task which is used
+to gate the release. If the EC task fails the release should be blocked. This
+functionality is handled by the Release Pipeline.
+
+For more information, see [HACBS Release
+Bundles](https://github.com/hacbs-release/release-bundles).
+
+_TODO: What about the post-build EC task?_
+
+### EC Renovate Bot
+
+One of the checks that EC does is to confirm the task definitions used in the
+build pipeline were defined in known and trusted task bundles. The purpose of
+the Renovate Bot is to automate the process for keeping such a list up to
+date.
+
+It's expected that a GitHub Action, or similar automation would automatically
+produce a pull request with the required update to the list when a new trusted
+task bundle is created.
```

Comment:
@lcarva I'm not sure I explained this well since I'm a little fuzzy on how it works. Lmk if you have some ideas how to describe this better.

Reply:
Could also be something to think about and iterate on later...